previews/azure-functions-service-bus.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Azure Functions with Azure Service Bus
+component: ASBFunctions
+summary: Azure Functions Preview
+related:
+ - samples/previews/azure-functions
+reviewed: 2020-07-06
+---
+
+Host NServiceBus endpoints with [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/).
```

Comment:
```suggestion
Host NServiceBus endpoints with [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/) and [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/) triggers.
```

(maybe `bindings` instead of `triggers`?)